demisto_sdk/commands/common/content/objects/pack_objects/__init__.py

```diff
@@ -11,6 +11,7 @@
 from .classifier.classifier import *  # lgtm [py/polluting-import]
 from .connection.connection import *  # lgtm [py/polluting-import]
 from .contributors.contributors import *  # lgtm [py/polluting-import]
+from .corrlation_rule.correlation_rule import *  # lgtm [py/polluting-import]
```

Comment:
Please import only what is needed and not the whole file. 
Learn from the mistakes of your ancestors.

`from .corrlation_rule.correlation_rule import CorrelationRule`.

Do that for the rest of the file.
Thank you!